MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -58,8 +58,13 @@ Please see [Before Development](MODULE_EVALUATION_TEMPLATE#before-development) f
   * _This is not applicable to libraries_
 * Must not depend on a FOLIO library that has not been approved through the TCR process
 * Gracefully handles the absence of third party systems or related configuration. (3, 5, 12)
-* Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication (6); and any disabled or intentionally ignored rules/recommendations are reasonably justified.
+* An automated code scannner hasn't identified any security issues, major code smells or excessive duplication (6); and any disabled or intentionally ignored rules/recommendations are reasonably justified.
   * See [Rule Customization](https://dev.folio.org/guides/code-analysis/#rule-customization) details. 
+  * The code scanner may be Sonar (via Jenkins), github Actions or other. The results of the code scanning need to be reproducible by the Community.
```

Comment:
My understanding is that GitHub actions are a build tool rather than a static code analyser

If that is the case, then it is similar to Jenkins rather than Sonar

At the moment, this sentence suggests to me that GitHub actions are an alternative to Sonar rather than an alternative to Jenkins